atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java

```diff
@@ -120,10 +125,18 @@ public void putWithTimestamps(TableReference tableRef, Multimap<Cell, Value> cel
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest request) {
```

Comment:
Super nit -- why not keep it consistent with the overridden method and call it range?